tests/manage/monitoring/prometheusmetrics/test_monitoring_defaults.py

```diff
@@ -239,3 +188,85 @@ def test_monitoring_reporting_ok_when_idle(workload_idle):
     assert mon_validation, mon_msg
     osds_msg = "ceph_osd_{up,in} metrics should indicate no OSD issues"
     assert all(osd_validations), osds_msg
+
+
+@skipif_managed_service
+@skipif_ocs_version("<4.6")
+class TestCephMonitoringAvailable:
+    @pytest.fixture(scope="class")
```

Comment:
This could be applicable for more tests. It could be session scope and in `/tests/conftest.py`

Reply:
done